src/python/pants/backend/python/util_rules/pex_test.py

```diff
@@ -363,10 +374,11 @@ def create_pex_and_get_all_data(
 def create_pex_and_get_pex_info(
     rule_runner: RuleRunner,
     *,
-    requirements=PexRequirements(),
-    entry_point=None,
-    interpreter_constraints=PexInterpreterConstraints(),
-    platforms=PexPlatforms(),
+    pex_type: type[Pex | VenvPex] = Pex,
+    requirements: PexRequirements = PexRequirements(),
+    entry_point: Optional[str] = None,
```

Comment:
No need to burn a tree, but FYI this can be `str | None`.

Reply:
Yeah, I won't burn a tree. Not happy at all with the Option Union thrash in the language.